SECURITY-INSIGHTS.yml

```diff
@@ -1,122 +1,349 @@
 header:
```

Comment:
I wonder if the base `SECURITY-INSIGHTS.yml` should be in the community repo, and the other repos extend it and list their maintainers (like what @Siddhant-K-code did in [the CLI PR](https://github.com/openfga/cli/pull/544/files)). Listing everyone as maintainer on this repo is inaccurate. WDYT @aaguiarz?